lms/views/lti/basic_launch.py

```diff
@@ -94,6 +81,15 @@ def unconfigured_launch(self):
 
         This happens if we cannot resolve a document URL for any reason.
         """
+
+        if not self.request.has_permission(Permissions.LTI_CONFIGURE_ASSIGNMENT):
```

Comment:
I reckon this is much nicer to read and understand in one go rather than two separate view + predicates.